src/core/layout/qgslayoutitemlegend.h

```diff
@@ -436,8 +503,18 @@ class CORE_EXPORT QgsLayoutItemLegend : public QgsLayoutItem
 
     void finalizeRestoreFromXml() override;
 
+    /**
+     * Creates an expression context
+     * \since QGIS 3.8
+     */
```

Comment:
No need to add docstrings to overridded methods like here (unless of course the docstrings add context specific additions)

Reply:
Removed.